source/guides/azure/infiniband.md

```diff
@@ -0,0 +1,493 @@
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+|   4  Tesla V100-SXM2...  On   | 00000005:00:00.0 Off |                    0 |
+| N/A   35C    P0    41W / 300W |      4MiB / 32768MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+|   5  Tesla V100-SXM2...  On   | 00000006:00:00.0 Off |                    0 |
+| N/A   36C    P0    43W / 300W |      4MiB / 32768MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+|   6  Tesla V100-SXM2...  On   | 00000007:00:00.0 Off |                    0 |
+| N/A   37C    P0    44W / 300W |      4MiB / 32768MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+|   7  Tesla V100-SXM2...  On   | 00000008:00:00.0 Off |                    0 |
+| N/A   38C    P0    44W / 300W |      4MiB / 32768MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+
++-----------------------------------------------------------------------------+
+| Processes:                                                                  |
+|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
+|        ID   ID                                                   Usage      |
+|=============================================================================|
+|    0   N/A  N/A      1396      G   /usr/lib/xorg/Xorg                427MiB |
+|    0   N/A  N/A      1762      G   /usr/bin/gnome-shell               16MiB |
+|    1   N/A  N/A      1396      G   /usr/lib/xorg/Xorg                  4MiB |
+|    2   N/A  N/A      1396      G   /usr/lib/xorg/Xorg                  4MiB |
+|    3   N/A  N/A      1396      G   /usr/lib/xorg/Xorg                  4MiB |
+|    4   N/A  N/A      1396      G   /usr/lib/xorg/Xorg                  4MiB |
+|    5   N/A  N/A      1396      G   /usr/lib/xorg/Xorg                  4MiB |
+|    6   N/A  N/A      1396      G   /usr/lib/xorg/Xorg                  4MiB |
+|    7   N/A  N/A      1396      G   /usr/lib/xorg/Xorg                  4MiB |
++-----------------------------------------------------------------------------+
+```
+
+## InfiniBand Driver
+
+On Ubuntu 20.04
+
+```shell
+sudo apt-get install -y automake dh-make git libcap2 libnuma-dev libtool make pkg-config udev curl librdmacm-dev rdma-core
+sudo apt-get install -y libgfortran5 bison chrpath flex graphviz gfortran tk dpatch quilt swig tcl
+VERSION="5.8-1.0.1.1"
```

Comment:
I don't know how Azure handles MOFED versions, is that guaranteed to remain the same, or do we need to tell users to check the version available and upgrade this?

Reply:
If you have rdma-core on ubuntu 20.04 then there's no need for MOFED as well, right?

Reply:
`lsmod | grep nv_peer_mem` is used to verify the installation. If I don't have MOFED, that doesn't work. Is there a better way to verify installation with only rdma-core?

Reply:
You could `apt install ibverbs-utils` and run `ibv_devinfo`.

Reply:
Cool. I'll try this when I run through the setup again.